cns/logger/cnslogger_windows.go

```diff
@@ -0,0 +1,29 @@
+package logger
+
+import (
+	"github.com/Azure/azure-container-networking/zapetw"
+	"github.com/pkg/errors"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	// LogPath is the path where log files are stored.
+	LogPath         = ""
+	etwCNSEventName = "AzureCNS"
+)
+
+func GetPlatformCores(loggingLevel zapcore.Level, encoder zapcore.Encoder) (zapcore.Core, error) {
```

Comment:
is there a reason that this remains exported?

Reply:
No. Making it lower camel case.